traitsui/testing/tester/_ui_tester_register/qt4/_interaction_helpers.py

```diff
@@ -11,7 +11,9 @@
 from pyface.qt import QtCore, QtGui
 from pyface.qt.QtTest import QTest
 
-from traitsui.testing.tester._compat import check_key_compat
+from traitsui.testing.tester._ui_tester_register._compat import (
```

Comment:
Darn... I think I had meant to type `_ui_tester_registry`. Sorry :(
The good thing about a name this unique is that is easy to do find-and-replace-all later. (This reminds me of one of the tips for an intermediate state of refactoring is to name something obviously bogus but also extremely unique so that it is easy to change them everywhere.)
The difference is really inconsequential, so I'd leave this up to you if you'd want to change this again.

Reply:
I noticed that but then just proceeded anyway, I should've asked to confirm.  Like you said though this will be a simple find-and-replace-all, so I will do that in a quick separate PR